deploy/helm/kubecf/assets/operations/instance_groups/app-autoscaler.yaml

```diff
@@ -0,0 +1,1040 @@
+  value:
+    domain: eventgenerator.service.cf.internal
+    targets:
+    - deployment: {{ .Release.Name | quote }}
+      domain: bosh
+      instance_group: asmetrics
+      network: default
+      query: '*'
+- type: replace
+  path: /addons/name=bosh-dns-aliases/jobs/name=bosh-dns-aliases/properties/aliases/-
+  value:
+    domain: scalingengine.service.cf.internal
+    targets:
+    - deployment: {{ .Release.Name | quote }}
+      domain: bosh
+      instance_group: asactors
+      network: default
+      query: '*'
+- type: replace
+  path: /addons/name=bosh-dns-aliases/jobs/name=bosh-dns-aliases/properties/aliases/-
+  value:
+    domain: metricsgateway.service.cf.internal
+    targets:
+    - deployment: {{ .Release.Name | quote }}
+      domain: bosh
+      instance_group: asnozzle
+      network: default
+      query: '*'
+- type: replace
+  path: /addons/name=bosh-dns-aliases/jobs/name=bosh-dns-aliases/properties/aliases/-
+  value:
+    domain: metricsserver.service.cf.internal
+    targets:
+    - deployment: {{ .Release.Name | quote }}
+      domain: bosh
+      instance_group: asmetrics
+      network: default
+      query: '*'
+
+# Add autoscaler related releases.
+- type: replace
+  path: /releases/-
+  value:
+    name: app-autoscaler
+    version: {{ (index .Values.releases "app-autoscaler").version | quote }}
+    url: {{ include "kubecf.releaseURLLookup" (list .Values.releases "app-autoscaler") }}
+    stemcell: {{ include "kubecf.stemcellLookup" (list .Values.releases "app-autoscaler") }}
+- type: replace
+  path: /releases/-
+  value:
+    name: postgres
+    version: {{ .Values.releases.postgres.version | quote }}
+    url: {{ include "kubecf.releaseURLLookup" (list .Values.releases "postgres") }}
+    stemcell: {{ include "kubecf.stemcellLookup" (list .Values.releases "postgres") }}
+
+# Add autoscaler instance groups.
+- type: replace
+  path: /instance_groups/0:before
+  value:
+    name: postgres_autoscaler
+    persistent_disk: 20480 # 20GB
+    {{- if .Values.kube.storage_class }}
+    persistent_disk_type: {{ .Values.kube.storage_class }}
+    {{- end }}
+    azs:
+    - z1
+    instances: 1
+    stemcell: default
+    networks:
+    - name: default
+    jobs:
+    - name: postgres
+      release: postgres
+      properties:
+        databases: &database
+          sslmode: &sslmode verify-full
+          tls: &database_tls
+            ca: ((postgres_ca.certificate))
+            certificate: ((postgres_server.certificate))
+            private_key: ((postgres_server.private_key))
+          databases:
+          - name: autoscaler
+            tag: default
+          db_scheme: postgres
+          port: &postgresPort 5432
+          roles:
+          - name: postgres
+            password: "((database_password))"
+            tag: default
+          connection_config: &databaseConnectionConfig
+            max_open_connections: 100
+            max_idle_connections: 10
+            connection_max_lifetime: 60s
```

Comment:
I'm surprised that we don't have good default values for these.  But then, all of `connection_config` appears to… not actually exist?

Reply:
Same as `db_scheme`, all of the `connection_config` is used for autoscaler release.  Such as https://bosh.io/jobs/golangapiserver?source=github.com/cloudfoundry-incubator/app-autoscaler-release&version=3.0.0#p%3dautoscaler.binding_db_connection_config.

I moved the values of `connection_config` to the autoscaler jobs.